community_server/src/Model/Table/TransactionsTable.php

```diff
@@ -122,4 +123,147 @@ public function buildRules(RulesChecker $rules)
 
         return $rules;
     }
+    
+    public function sortTransactions($a, $b)
+    {
+        if ($a['date'] == $b['date']) {
+            return 0;
+        }
+        return ($a['date'] > $b['date']) ? -1 : 1;
+    }
```

Comment:
```suggestion
    public function sortTransactions($a, $b)
    {
        return $b['date'] - $a['date'];
    }
```

Not sure this works since it is not exactly -1 and 1, but that's the standard I believe.
See: https://stackoverflow.com/questions/10709803/java-comparator-how-to-sort-by-integer

Reply:
No, it isn't working, I have try it. I get this error message:
Object of class Cake\I18n\FrozenTime could not be converted to int 
In this function no integers are compared, Time Objects are compared and they havn't a minus operator.

Reply:
ok then lets leave it as that for now - tho converting it to unix format and doing the - would still be an improvement. Maybe you have the time to investigate at some point. Not needed for the this PR and off track

Reply:
In the new listTransactions the function isn't used at all so after updating the rest we can safely remove it.